packages/babel-helper-module-imports/src/import-injector.ts

```diff
@@ -439,6 +437,7 @@ export default class ImportInjector {
       });
 
       const targetPath = body.find(p => {
+        // @ts-expect-error todo(flow->ts): avoid mutations
```

Comment:
TS complains unused '@ts-expect-error' directive here.

Reply:
this is because of missing reference to @babel/traverse in tsconfig.json (because traverse types are missing - `p` instead of `NodePath` is typed as `any`, and so there is no error about accessing undeclared property on `p.node`)

made a fix in #12941